pkg/filesystem/subvolume.go

```diff
@@ -195,12 +203,14 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 					// check the state of the stale subvolume
 					// if it is snapshot-retained then skip listing it.
 					if state == "snapshot-retained" {
-						status = state
 						continue
 					}
-					// check if the stale subvolume has snapshots.
-					if checkSnapshot(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name) {
-						status = staleWithSnapshot
+					// check for snapshot only if the subvolume info doesn't return any error.
+					if err == nil {
```

Comment:
why we need this check here? i assume `err` should be handled above not here

Reply:
in case there is error running subvolume info command, we shouldn't check for the snapshot also,  that would just be an addition to  run time.

Reply:
okay, after adding continue to above case, we no more need this check as in case of err it will not go further. Thanks for notifying.